config/overlays/odh/patches/manager_auth_config_patch.yaml

```diff
@@ -0,0 +1,26 @@
+# This patch injects auth config properties from auth-refs configmap in the controller manager
+apiVersion: apps/v1
```

Comment:
don't see this referenced in kustomization.yaml, is that not needed?

Reply:
it is, line 18 of `config/overlays/odh/kustomization.yaml`